### ISSUE TITLE
(File.Error) could not read file VERSION: no such file or directory in 0.5.1

### ISSUE THREAD

**Issue by KazuCocoa** [hexedpackets/trot#18]
I tried with trot `0.5.1`, then the following error caused.

```
==> trot_example
Error while loading project :trot at /Users/myname/Documents/github/trot_example/deps/trot
** (File.Error) could not read file VERSION: no such file or directory
    (elixir) lib/file.ex:245: File.read!/1
    mix.exs:4: (module)
    (stdlib) erl_eval.erl:657: :erl_eval.do_apply/6
```

**Comment by hexedpackets** [comment]
Resolved with your PR #19, I'll update the hex package with the fix.

**Comment by KazuCocoa** [comment]
Thanks for your quick updating !
trot 0.5.2 works well with plug v0.13.0 :)
